### ISSUE TITLE
Message misplacement

### ISSUE THREAD

**Issue by quaeched** [MCMrARM/revolution-irc#153]
Messages sent to users are sometimes written into "SERVER" dialog, not the actual chat.

**Comment by MCMrARM** [comment]
Huh? A screenshot please.

**Comment by quaeched** [comment]
just noticed that the conversation ("M3TRIKX-") was automatically removed after messaging the other user... maybe it "wasn't really there" and the text was passed to the "only present" conversation - the "SERVER"?

**Comment by quaeched** [comment]
after I restart the application the tabs are gone

**Comment by quaeched** [comment]
when I message the user now, the messages (the two) are not there

**Comment by quaeched** [comment]
btw, is there any other way to reenvoke the tab without messaging the user?

**Comment by MCMrARM** [comment]
Yes, using the Message user/Join channel with the user nick instead of channel feature in the menu

**Comment by quaeched** [comment]
great thank You. though I noticed that /join user doesn't work but I guess that is an IRC thing? unfortunately the <b>Message user</b> option doesn't work for me (don't know if it's a submitted issue) - I am unable to write into the body text - after I write the username I can't get any further (clicking on the arrow doesn't work, neither does clicking on the <b>Send a message</b> field)...
